index.js

```diff
@@ -77,31 +131,27 @@ const getAllUsers = async () => {
         }))
 }
 
-const delay = async(t) => {
-    return new Promise(function(resolve) { 
-        setTimeout(resolve, t)
-    });
- }
+const delay = duration => new Promise(resolve => setTimeout(resolve, duration));
```

Comment:
Yay 2015

Reply:
are you even a JS dev if you don't use arrow functions